### ISSUE TITLE
test_meshing_polyhedron sometimes fails

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal#1507]
@cjamin 

In [CGAL-4.10-Ic-8, with the Intel Compiler 17][*], the test `test_meshing_polyhedron` fails to run in the parallel mode: the number of elements is too different. That error pops up from time to time, in different columns. The compiler is not important: that probably depends on the seed only.

I have had already reported that issue several time. You told me that you do not want to modify the bounds of intervals of number of elements, because you still have to find out by the meshing in parallel run differently from the sequential meshing. But that was at least a year ago. Now I create a real issue entry to track the progress on that matter.

[*]: https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.10-Ic-8/Mesh_3/TestReport_lrineau_x86-64_Linux-Fedora_IntelCompiler-17-with-assertions.gz

**Comment by cjamin** [comment]
Was it a discussion by e-mail? I can't find it and I don't remember it.
I only found [this discussion](https://github.com/CGAL/cgal/pull/559#issuecomment-166253704) but it seems I missed it (most likely because you did not use "@cjamin").

Actually, as you can see in the code, most of the bounds are already different in sequential and in parallel. This is not a bug: the parallel algorithm treats the elements in an order that is more "fuzzy" than in sequential mode. In most cases, the difference is not very big, but it is random, so I'm not surprised that some cases are worse than others.
To sum up: it is normal to have to specify wider intervals, and it's already the case in most tests.

I suggest to replace 350 by 300, and 420 by 450. If you agree, I can take care of it.

**Comment by lrineau** [comment]
OK, thanks.

**Comment by lrineau** [comment]
Fixed by #1508.
